app/models/organization.rb

```diff
@@ -9,28 +10,35 @@ def initialize(init_params)
     @name = init_params['name']
     @description = init_params['description']
     @imageUrl = init_params['profileImageUrl']
-    @categories = init_params['type']
-    @interests = init_params['categories']
+    @categories = init_params['categories']
+    @interests = init_params['interests']
   end
 
   def self.get_organizations(num, categories, interests)
     orgs = self.get_orgs_from_api()
-
+    csv_text = File.read('lib/interest.csv')
+    interest = CSV.parse(csv_text, headers: true)
     top_orgs = []
     orgs.each do |i|
       org_categories = i['categories']
       org_categories.each do |c|
         categories.each do |quiz_category|
           if c['categoryName'].downcase.include? quiz_category.downcase
+            org_interest = interest.find {|row| row['Organization ID'].to_i == i['organizationId']}
+            if org_interest != nil

```

Comment:
This was done to deal with the case of interest.find {|row| row['Organization ID'].to_i == i['organizationId']} returning nil

Reply:
interest.find doesn't return a boolean, it returns either the element of the array if it successfully found something, or nil. The if condition is fine as it is.